HealthcareSystem/PatientWebApp/Controllers/PatientController.cs

```diff
@@ -175,7 +175,19 @@ public IActionResult GetMaliciousPatients()
             }
         }
 
-
+        [HttpGet("{jmbg}/canceled-examinations")]
+        public int GetNumberOfCanceledExaminations(string jmbg)
+        {
+            try
+            {
+                int number = _patientService.GetNumberOfCanceledExaminations(jmbg);
+                return number;
+            }
+            catch (DatabaseException exception)
+            {
+                return -1;
```

Comment:
Why return -1 instead of an error code?

Reply:
I repaired the mistake, it should be ok. I will commit it now.

Reply:
Fixed in 9ea7efda4c6eecdac1d8252df92ef7f13bacc7d1